api/database/migrations/2023_07_19_140947_create_digital_contracting_questionnaires_table.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Query\Expression;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // main table for digital contracting questionnaires
+        Schema::create('digital_contracting_questionnaires', function (Blueprint $table) {
+            $table->uuid('id')->primary('id')->default(new Expression('gen_random_uuid()'));
+            $table->timestamps();
+            $table->boolean('read_preamble')->nullable();
+            $table->uuid('department_id')->nullable();
+            $table->foreign('department_id')->references("id")->on("departments");
+            $table->string('department_other')->nullable();
+            $table->string('branch_other')->nullable();
```

Comment:
I'm a little confused about the "Branch" field. The model says "influenced by department / agency" and that it is a combobox. Will this be a new field that is added at a later date and for now we are only storing the "other" free-form text field value? 🤔

Reply:
That was based on this comment in Figma:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/106897e7-6fea-4949-bbee-e8bcf0a4fe3b)
I imagine we might add a branch_id field in the future.